airbyte-integrations/connectors/source-google-ads/acceptance-test-config.yml

```diff
@@ -22,20 +22,22 @@ acceptance_tests:
           path: "integration_tests/expected_records.jsonl"
         timeout_seconds: 600
         empty_streams:
-          - name: "accounts"
-            bypass_reason: "Floating data"
           - name: "display_topics_performance_report"
             bypass_reason: "Stream not filled yet."
           - name: "account_labels"
             bypass_reason: "Unable to seed the stream"
-          - name: "ad_group_criterion_labels"
-            bypass_reason: "Unable to seed the stream"
           - name: "click_view"
             bypass_reason: "Stream not filled yet."
           - name: "unhappytable"
             bypass_reason: "Stream not filled yet."
           - name: "shopping_performance_report"
             bypass_reason: "Stream not filled yet."
+        ignored_fields:
+          accounts:
+            - name: customer.optimization_score_weight
+              bypass_reason: "can be updated, also it is sometimes integer, sometimes float"
```

Comment:
can manually validate typing and update it if needed?

Reply:
Updated